### PR TITLE
fix(color): Improve the library size for webpack

### PR DESCRIPTION
**Pull request by iamakulov**
Follow-up after https://twitter.com/b_hough/status/878015824056942592

### CODE REVIEW

**Comment by iamakulov** [comment]
There’s something wrong with 9a57c85. Will try to re-generate the assets now

**Comment by iamakulov** [comment]
Done.

File: docs/docs/index.html [hunk 1]
```diff
@@ -845,7 +845,7 @@ <h3 class='fl m0' id='clearfix'>
     </h3>
     
     
-      <a class='fr fill-darken0 round round pad1x quiet h5' href='https://github.com/styled-components/polished/blob/c5ccb3bf1c210d26eb3c2a3b7ab71a817b49e7b3/src/mixins/clearFix.js#L26-L35'>
+      <a class='fr fill-darken0 round round pad1x quiet h5' href='https://github.com/iamakulov/polished/blob/9a57c85ee212dd49b3736dcd65b49d75c0a3ea0c/src/mixins/clearFix.js#L26-L35'>
```
Comment (iamakulov) [inline docs/docs/index.html]: Not sure about these changes. It that because of a different `origin`? Should I fix it?

Comment (iamakulov) [inline docs/docs/index.html]: Never mind, fixed that.

**Comment by iamakulov** [comment]
OK, now looks like really done. Tests on the last commit errored because yarn failed to download, should be fine after re-launch.

**Comment by terales** [comment]
Here is a link to case study: https://iamakulov.com/notes/polished-webpack/

**Comment by Siyfion** [comment]
Great article and fantastic work @iamakulov!

**Comment by nikgraf** [comment]
Wow, really great article and good work on the PR @iamakulov ✨

**Pull request merged**